internal/support/notifications/router.go

```diff
@@ -68,5 +68,11 @@ func LoadRestRoutes() *mux.Router {
 	// /api/v1/ping
 	b.HandleFunc("/ping", pingHandler).Methods(http.MethodGet)
 
+	// Configuration
+	b.HandleFunc("/config", configHandler).Methods(http.MethodGet)
+
+	// Metrics
+	b.HandleFunc("/metrics", metricsHandler).Methods(http.MethodGet)
+
```

Comment:
Move these and also the /ping assignment up above line 26. Since you will be adding these routes to the services, you will also need to add them to the respective clients since the SysMgmt will call the services through a client rather than inline. For this PR, modify the /pkg/clients/notifications/client.go so that these routes can be called. You should also define constants for these routes in /pkg/clients/constants.go. The above function calls will then look like:

`b.HandleFunc(clients.ApiConfigRoute, configHandler).Methods(http.MethodGet)`

You will then use the constants on subsequent PRs when defining the routes in other services.

Reply:
Done for /config. Will do so shortly for /metrics.